src/pageTypes/views/PageTypeUpdate.tsx

```diff
@@ -0,0 +1,163 @@
+import Button from "@material-ui/core/Button";
+import { attributeUrl } from "@saleor/attributes/urls";
+import NotFoundPage from "@saleor/components/NotFoundPage";
+import { WindowTitle } from "@saleor/components/WindowTitle";
+import useBulkActions from "@saleor/hooks/useBulkActions";
+import useNavigator from "@saleor/hooks/useNavigator";
+import useNotifier from "@saleor/hooks/useNotifier";
+import { commonMessages } from "@saleor/intl";
+import { usePageTypeUpdateMutation } from "@saleor/pageTypes/mutations";
+import createMetadataUpdateHandler from "@saleor/utils/handlers/metadataUpdateHandler";
+import {
+  useMetadataUpdate,
+  usePrivateMetadataUpdate
+} from "@saleor/utils/metadata/updateMetadata";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import PageTypeDetailsPage, {
+  PageTypeForm
+} from "../components/PageTypeDetailsPage";
+import { usePageTypeDetailsQuery } from "../queries";
+import { pageTypeListUrl, pageTypeUrl, PageTypeUrlQueryParams } from "../urls";
+
+interface PageTypeUpdateProps {
+  id: string;
+  params: PageTypeUrlQueryParams;
+}
+
+export const PageTypeUpdate: React.FC<PageTypeUpdateProps> = ({ id }) => {
+  const navigate = useNavigator();
+  const notify = useNotifier();
+  const attributeListActions = useBulkActions();
+  const intl = useIntl();
+  const [errors, setErrors] = React.useState({
+    addAttributeErrors: [],
+    editAttributeErrors: [],
+    formErrors: []
+  });
```

Comment:
TBH I don't know why it exists, not here, not in product types.
Can we remove it or something happens?

I believe it's a leftover code from time when attributes were created in product type details view.

Reply:
sure, removed